gcc-4.8/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
 
 build:
   detect_binary_files_with_prefix: true # [not linux32]
-  number: 1
+  number: 3
 
```

Comment:
Bumped build number to 3 because conda's `gcc` package already has build number=2.